network/src/commonMain/kotlin/com/wire/kalium/network/utils/NetworkUtils.kt

```diff
@@ -199,12 +200,24 @@ internal suspend fun handleUnsuccessfulResponse(
     result: HttpResponse
 ): NetworkResponse.Error {
     val status = result.status
+    val bodyText = try {
+        result.bodyAsText()
```

Comment:
bodyAsText will call .body() and .body() can cause DoubleReceiveException, this is why first read response as string and then parse it